app/app/settings.py

```diff
@@ -166,13 +166,21 @@
 OSU_CLIENT_ID = int(os.getenv("OSU_CLIENT_ID"))
 OSU_CLIENT_SECRET = os.getenv("OSU_CLIENT_SECRET")
 OSU_REDIRECT_URL = os.getenv("OSU_REDIRECT_URL")
-OSU_LOGIN_URL = AuthHandler(
+OSU_DEV_SERVER = bool(int(os.getenv("OSU_DEV_SERVER")))
+
+AUTH = AuthHandler(
```

Comment:
picky, but I would make this lowercase or prefix it with osu

Reply:
lowercased